app/build.gradle

```diff
@@ -40,12 +40,18 @@ android {
             signingConfig signingConfigs.config
         }*/
     }
-
+    
+    android.buildTypes.each { type ->
+        type.buildConfigField 'String', 'RC_MAPS_API_KEY', RC_MAPS_API_KEY
+    }
 
     repositories {
         maven {
             url "https://raw.github.com/dimelo/Dimelo-Android/master"
         }
+        maven {
+                url "https://raw.github.com/dimelo/engage-digital-messaging-android-location-source/master"
```

Comment:
correct me if I'm wrong but the link should be `https://raw.github.com/ringcentral/engage-digital-messaging-android-location/master`, right?

Reply:
https://raw.github.com/name of repos ,

and the name of the repo is engage-digital-messaging-android-location, dont need to ringCentral

Reply:
dimelo is our old account that has been closed we're now using the ringcentral one, the only reason it works is because dimelo account has been configured to redirect to the ringcentral one automatically so we should definitely use the ringcentral link

Reply:
ok I will channge dimelo by ringcentral